raw_tiles/source/wikidata.sql

```diff
@@ -1,23 +1,33 @@
-SELECT
-  w.id AS id,
-  hstore_to_json(w.tags) AS properties
-FROM wikidata w
-JOIN
-  (
-    SELECT DISTINCT tags->'wikidata' AS wikidata
-    FROM planet_osm_point
-    WHERE way && {{box}} AND tags ? 'wikidata'
+WITH wikidata_ids AS (
+  SELECT DISTINCT tags->'wikidata' AS wikidata
+  FROM planet_osm_point
+  WHERE way && {{box}} AND tags ? 'wikidata'
```

Comment:
nit: the `way` here is actually a `node`

Reply:
Since polygons are sometimes ways/relations `way` is general stand-in for "map feature". So it's fine, or use more generic term everywhere?

Reply:
Could say `osm_pp` and `ne_pp`...

Reply:
Unfortunately, `way` is the name of the column that `osm2pgsql` uses, even for the points table.

This query isn't pulling in the "populated place" data from OSM, it just gets `fclass_*` from the NE tables, so I think an `osm_pp` label would be misleading here.

Reply:
Changed to `ne_pp` in 44c4e9d.